docs/conf/inspircd.conf.example

```diff
@@ -177,11 +177,13 @@
 # module).
 #<bind address="" port="7002" type="clients" hook="websocket">
 
-# When linking servers, the OpenSSL and GnuTLS implementations are completely
-# link-compatible and can be used alongside each other
-# on each end of the link without any significant issues.
-# Supported SSL types are: "openssl" and "gnutls".
-# You must load the ssl_openssl module for OpenSSL or ssl_gnutls for GnuTLS.
+# When linking servers, the OpenSSL, GnuTLS, and mbedTLS implementations are
+# completely link-compatible and can be used alongside each other on each end
+# of the link without any significant issues.
+# Supported SSL types are:
+#   "openssl" (requires the ssl_openssl module)
+#   "gnutls" (requires the ssl_gnutls module)
+#   "mbedtls" (requires the ssl_mbedtls module)
```

Comment:
This is not documented anywhere currently, the documentation of SSL modules in the config files is essentially "check out the wiki for the available config options" in 2.0. Perhaps we should keep that and show a basic configuration with a ssl profiles.